ui/language_utils.js

```diff
@@ -199,6 +199,18 @@ shaka.ui.LanguageUtils = class {
     // When there is a loss of specificity (either to a base language or to
     // "unknown"), we should append the original language code.
     // Otherwise, there may be multiple identical-looking items in the list.
+    if (window.Intl && 'DisplayNames' in Intl) {
```

Comment:
You could put this above line 194, right? Since we don't use the value of `language` that we get from that line.
That would also give us room for a comment describing what `Intl.DisplayNames` is, and why we are using it preferentially.

Reply:
Done!